Update the Alerting HA docs (#60292)
grafana/grafana: docs/sources/alerting/set-up/configure-high-availability/_index.md

# Enable alerting high availability

> You can enable alerting high availability support by updating the Grafana configuration file, additional steps are required to make this work if you run Grafana in a Kubernetes cluster. Both setups are described in this tutorial.

Reviewer: ```suggestion
There are two ways of enabling alerting high availability depending on whether you use Grafana or if you run Grafana in a Kubernetes cluster.
```

Reviewer: Ok:) So:
You can enable alerting high availability support by updating the Grafana configuration file. If you run Grafana in a Kubernetes cluster, additional steps are required. Both options are described below.

Author: Sounds good!

> ## In Grafana

Reviewer: ```suggestion
## Enable alerting high availability in Grafana
```

3. Set `[ha_listen_address]` to the instance IP address using a format of `host:port` (or the [Pod's](https://kubernetes.io/docs/concepts/workloads/pods/) IP in the case of using Kubernetes).
   By default, it is set to listen to all interfaces (`0.0.0.0`).

> ## Using Kubernetes

Reviewer: ```suggestion
## Enable alerting high availability using Kubernetes
```

    fieldRef:
      fieldPath: status.podIP
```

1. Add the port 9094 to the Grafana deployment:

  ```yaml
  ports:
  - containerPort: 3000
    name: http-grafana
    protocol: TCP
  - containerPort: 9094
    name: grafana-alert
    protocol: TCP
  ```

> 2. And the environment variables to the Grafana deployment:

Reviewer: ```suggestion
2. Add the environment variables to the Grafana deployment:
```

  ```yaml
  env:
    - name: POD_IP
      valueFrom:
        fieldRef:
          fieldPath: status.podIP
  ```

> 3. Create a headless service which is returning the pod IP instead of the service IP, which is what the `ha_peers` need:

Reviewer: ```suggestion
3. Create a headless service that returns the pod IP instead of the service IP, which is what the `ha_peers` need:
```

  ```yaml
  apiVersion: v1
  kind: Service
  metadata:
    name: grafana-alerting
    namespace: grafana
    labels:
      app.kubernetes.io/name: grafana-alerting
      app.kubernetes.io/part-of: grafana
  spec:
    type: ClusterIP
    clusterIP: "None"
    ports:
      - port: 9094
    selector:
      app: grafana
  ```

4. Make sure your grafana deployment has the label matching the selector, e.g. `app:grafana`.

> 5. Now add in the grafana.ini:

Reviewer: ```suggestion
5. Add in the grafana.ini:
```
